modules/must-gather-flags.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="must-gather-flags_{context}"]
+= Must-gather flags
+
+The flags listed in the following table are available to use with the `oc adm must-gather` command.
+
+.{product-title} flags for `oc adm must-gather`
+[cols="1,1,3",options="header"]
+|====
+|Flag |Example command |Description
+
+|`--all-images`
+|`oc adm must-gather --all-images=false`
+|Collect `must-gather` data using the default image for all Operators on the cluster that are annotated with `operators.openshift.io/must-gather-image`.
+
+|`--dest-dir`
+|`oc adm must-gather --dest-dir='<directory_name>'`
+|Set a specific directory on the local machine where the gathered data is written.
+
+|`--host-network`
+|`oc adm must-gather --host-network=false`
+|Run `must-gather` pods as `hostNetwork: true`. Relevant if a specific command and image needs to capture host-level data.
+
+|`--image`
+|`oc adm must-gather --image=[<plugin_image>]`
+|Specify a `must-gather` plugin image to run. If not specified, {product-title}'s default `must-gather` image will be used.
+
+|`--image-stream`
+|`oc adm must-gather --image-stream=[<image_stream>]`
+|Specify an image stream (namespace/name:tag) containing a `must-gather` plugin image to run.
+
+|`--node-name`
+|`oc adm must-gather --node-name='<node>'`
+|Set a specific node to use. If not specified, by default a random master is used.
+
+|`--node-selector`
+|`oc adm must-gather --node-selector='<node_selector_name>'`
+|Set a specific node selector to use. Only relevant when specifying a command and image which needs to capture data on a set of cluster nodes simultaneously.
+
+|`--run-namespace`
+|`oc adm must-gather --run-namespace='<namespace>'`
+|An existing privileged namespace where `must-gather` pods should run. If not specified, a temporary namespace is generated.
+
+|`--since`
+|`oc adm must-gather --since=<time>`
+||Only return logs newer than the specified duration. Defaults to all logs. Plugins are encouraged but not required to support this. Only one `since-time` or `since` may be used.
```

Comment:
```suggestion
|Only return logs newer than the specified duration. Defaults to all logs. Plugins are encouraged but not required to support this. Only one `since-time` or `since` may be used.
```
table is broken here in the preview because of the extra column marker